### ISSUE TITLE
Adapt to recent USDB changes

### ISSUE THREAD

**Issue by bohning** [bohning/usdb_syncer#96]
- [x] USDB string translations (German, French)
- [x] remove URL escape for colons (now everything after the first colon in a header tag is considered the value of that tag, so colons can be used in URLs without escaping them - so finally discogs cover URLs can be used directly! Only commas and equal signs need to be escaped due to their special meaning for meta tags)
- [x] add logic to support fully qualified URLs with protocol (https://, http://), which are now supported (see above)
- [x] USDB now supports uploading UTF8 files (but unfortunately sends them out as CP1252 when the user hits the download button)
- [x] login via Firefox browser cookies not working? (possibly access permission problems?)
- [x] maybe: remove hard cap for USDB songs to fetch (currently 30.000)
- [x] remove support for `co-protocol` and`bg-procotol` (see above)
- [x] fetch USDB cookie only once (cache)
- [x] maybe: login via USDB user/pass (store via [keyring](https://pypi.org/project/keyring/)?)

~maybe: add progress bar to song list fetch (N pages of 100 songs each)~ (deferred)

**Comment by RumovZ** [comment]
> login via Firefox browser cookies not working? (possibly access permission problems?)

Works for me now. :) Not sure if I did something wrong or the dependency fixed, but I'm marking this as resolved.

**Comment by RumovZ** [comment]
It seems the golden notes column on the USDB song list is now localized, so the syncer now always parses golden notes as `False` if the user's language is not English.

**Comment by bohning** [comment]
> It seems the golden notes column on the USDB song list is now localized, so the syncer now always parses golden notes as `False` if the user's language is not English.

This should be fixed by #123 .

**Comment by RumovZ** [comment]
Another one: The song list is now only available if you're logged in. That means if a user opens the syncer for the first time, they won't see any songs. Maybe we should add some sort of welcome screen that instructs them how to setup the app.
We could also try to detect the used browser or at least use some common default instead of `NONE`.

Or we could ship a song list with the binaries (about 4 MB, 0.7 MB zipped). Then users wouldn't even have to do a full sync at any point.

**Comment by bohning** [comment]
> Another one: The song list is now only available if you're logged in. That means if a user opens the syncer for the first time, they won't see any songs. Maybe we should add some sort of welcome screen that instructs them how to setup the app. We could also try to detect the used browser or at least use some common default instead of `NONE`.

It is also possible to fetch the respective cookie from all supported browsers (from: https://github.com/borisbabic/browser_cookie3/blob/master/README.md)
Alternatively if you don't know/care which browser has the cookies you want then all available browser cookies can be loaded:
```
import browser_cookie3
cj = browser_cookie3.load()
```

> Or we could ship a song list with the binaries (about 4 MB, 0.7 MB zipped). Then users wouldn't even have to do a full sync at any point.
Good idea, I like.
